path_utils_test.go

```diff
@@ -12,124 +12,31 @@ func TestPatternMatches(t *testing.T) {
 	wd, err := os.Getwd()
 	require.NoError(t, err)
 
-	wdParts := SplitPath(wd)
-	wdLen := len(wdParts)
-
 	testCases := []struct {
 		path     string
-		prefix   int
 		expected []string
 	}{
-		{"cmd", wdLen + 1, []string{
+		{"cmd", []string{
 			filepath.Join(wd, "cmd"),
 		}},
-		{"cmd/*", wdLen + 1, []string{
+		{"cmd/*", []string{
 			filepath.Join(wd, "cmd/gitbase"),
 		}},
-		{"cmd/gitbase/*", wdLen + 2, []string{
+		{"cmd/gitbase/*", []string{
 			filepath.Join(wd, "cmd/gitbase/command"),
 			filepath.Join(wd, "cmd/gitbase/main.go"),
 		}},
-		{"cmd/../cmd/gitbase/*", wdLen + 2, []string{
+		{"cmd/../cmd/gitbase/*", []string{
 			filepath.Join(wd, "cmd/gitbase/command"),
 			filepath.Join(wd, "cmd/gitbase/main.go"),
 		}},
 	}
 
 	for _, test := range testCases {
 		t.Run(test.path, func(t *testing.T) {
-			prefix, files, err := PatternMatches(test.path)
+			files, err := PatternMatches(test.path)
 			require.NoError(t, err)
-			require.Equal(t, test.prefix, prefix)
 			require.Exactly(t, test.expected, files)
 		})
 	}
 }
-
-func TestPatternPrefixDepth(t *testing.T) {
```

Comment:
why did you delete that test?

Reply:
Because the function was tested here is not needed now so I removed it.